packages/snage/src/config/type.ts

```diff
@@ -65,9 +65,14 @@ export interface Sort {
     order: 'asc' | 'desc';
 }
 
+export interface RawLink {
+    name: string;
+    link: string;
+}
 export interface Link {
     name: string;
     link: string;
+    requiredFields: string[];
 }
```

Comment:
Wäre so etwas denkbar?
```suggestion
export interface Link {
    name: (values: NoteValues) => O.Option<string>;
    link: (values: NoteValues) => O.Option<string>;
}
```

oder vielleicht noch exakter

```suggestion
export interface LinkProvider {
    (values: NoteValues): O.Option<[string, string]>;
}
```